package.json

```diff
@@ -25,8 +24,8 @@
     "lint": "eslint ./",
     "lint:fix": "eslint --fix ./",
     "lint:tofile": "eslint --no-color -o ./eslint-report.log ./",
```

Comment:
add `lint:typescript` script that runs `tsc --noEmit` to check types (and run it as part of `check`